views/default/elements/forms/label.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Form input label view
+ *
+ * @uses $vars['label'] Label text
```

Comment:
Another alternative to allow either string or an array with 'text' parameter and other attributes to pass to the label tag